src/main/kotlin/no/nav/dagpenger/regel/api/models/InntektResponse.kt

```diff
@@ -16,3 +16,17 @@ data class InntektResponse(
         }
     }
 }
+
+data class InntektResponseGrunnlag(
+    val inntekt: BigDecimal,
+    val periode: Int, // todo: enum?
+    val inntektsPeriode: InntektsPeriode,
+    val inneholderFangstOgFisk: Boolean
+) {
```

Comment:
I think you should use an enum instead of this validation logic

Reply:
Vi vet det, men tar det i en annen branch.